css-box-model/styles.css

```diff
@@ -0,0 +1,11 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  border: 0;
```

Comment:
Remove this line of code, the buttons should have a border using the default styling that gets applied by the browser